src/main/java/pl/byczazagroda/trackexpensesappbackend/model/Wallet.java

```diff
@@ -0,0 +1,54 @@
+package pl.byczazagroda.trackexpensesappbackend.model;
+
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Pattern;
+import javax.validation.constraints.Size;
+import java.io.Serial;
+import java.io.Serializable;
+import java.time.Instant;
+
+/**
+ * Entity class for money wallet.
+ *
+ * @author DawidStuzynski
+ * @version 1.0
+ * @since track-expenses-app-backend 2.0
+ */
```

Comment:
This is a wrong versions. It should be 0.2.0 in both cases. However I would suggest to keep only @Since tag which "only identifies a time point in the past at which new changes or features came into existence". This class will be changing a couple of times because of add new attributes. So i can be problematic maintain the correct version for the version tag. I suggest live only since tag with 0.2.0.  I dont't know what Dawid told you, but I think we should be consistent and when  we add Javadoc to one public class we should do to the rest of public class and public fields.

Reply:
As @Iwona007 said, you should change versions.